Mathlib/Order/ScottContinuity.lean

```diff
@@ -0,0 +1,68 @@
+/-
```

Comment:
Why not put Scott Continuity in `Topology/Order/ScottTopology` along with `DirSupInacc` and `DirSupClosed`?

Reply:
Do as you want. I just got annoyed editing `Order.Bounds.Basic`.

I must say it would be a bit surprising to have that under `Topology` given that that part of the theory is topology-free, But who am I to decide?